### PR TITLE
Add github workflow to update files from GEDCOM repo

### PR DESCRIPTION
**Pull request by dthaler**
Currently this workflow updates the master branch when the GEDCOM repository's main branch is updated.  If we instead only want to do it when we do a "release", it's easy to update the workflow to do that instead, by using (say) a tag in the GEDCOM repository.

However, I think it would be useful to generate a live update even if it's in a separate branch of GEDCOM.io so one can see the latest unreleased copy.

### CODE REVIEW

File: .github/workflows/update-files.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# This action will regenertae generated files on a weekly schedule.
```
Comment (dthaler) [inline .github/workflows/update-files.yml:1]:
```suggestion
# This action will regenerate generated files on a weekly schedule.
```

File: .github/workflows/update-files.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        export MD_FILES=$(ls ${{ env.SPECDIR }}gedcom*.md | sort)
+        python3 hyperlink.py ${MD_FILES} GEDCOM-tmp.md
+        ls -l GEDCOM-tmp.md
+
+    - name: Generate GEDCOM-tmp.html
+      working-directory: ${{github.workspace}}/GEDCOM/build
+      shell: sh
+      run: |
+        echo "Ready..."
+        echo "SPECDIR: ${SPECDIR}"
```
Comment (dthaler) [inline .github/workflows/update-files.yml:61]:
```suggestion
```

File: .github/workflows/update-files.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        export MODTIME=$(git log -1 --date="format:%_d %B %Y" --format="%ad" -- ${SPECDIR})
+        echo "MODTIME: ${MODTIME}"
```
Comment (dthaler) [inline .github/workflows/update-files.yml:63]:
```suggestion
```

File: .github/workflows/update-files.yml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        export PDARGS="--syntax-definition=gedcom.xml --syntax-definition=gedstruct.xml --syntax-definition=abnf.xml --from=markdown+smart --standalone --css=pandoc.css --template=template.html --toc --number-sections --self-contained --highlight-style=kate --wrap=none"
+        echo "PDARGS: ${PDARGS}"
```
Comment (dthaler) [inline .github/workflows/update-files.yml:65]:
```suggestion
```

**Comment by dthaler** [comment]
I'm thinking we should have 3 branches updated in GEDCOM.io:
* Doing a GEDCOM release updates GEDCOM.io "main" branch
* Any checkin to GEDCOM main updates GEDCOM.io "next-patch" branch
* Any checkin to GEDCOM next-minor updates GEDCOM.io "next-minor" branch

**Comment by dthaler** [comment]
This workflow now does (weekly):

* Update the main branch from the latest GEDCOM release version, if not already up to date
* Update the `next-patch` branch from the GEDCOM `main` branch, if not already up to date
* Update the `next-minor` branch from the GEDCOM `next-minor` branch, if not already up to date

**Review by lynchrs3 — APPROVED** [review]

**Review by tychonievich — APPROVED** [review]

**Pull request merged**